carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/DriverPool.java

```diff
@@ -68,15 +68,16 @@ public static WebDriver getDriver() {
     }
 
     /**
-     * Get first registered driver from Pool.
+     * @deprecated (Obsolete method which will be removed in 6.0 core generation. Standard getDriver(..) should be reused!)
      * 
+     * Get first registered driver from Pool.
      * @return default WebDriver
      */
     @Deprecated
     public static WebDriver getExistingDriver() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1133)